pom.xml

```diff
@@ -35,6 +36,8 @@
     </licenses>
 
     <modules>
+        <module>http-delegation</module>
```

Comment:
Do we want to add a Gradle equivalent to these two projects? Since they don't get built as part of the normal Jenkins run, they won't fail the build is a header is missing or incorrect.